clang/lib/AST/ByteCode/Context.h

```diff
@@ -67,6 +67,10 @@ class Context final {
   bool evaluateCharRange(State &Parent, const Expr *SizeExpr,
                          const Expr *PtrExpr, std::string &Result);
 
+  /// Evaluate \param E and if it can be evaluated to a sirint literal,
```

Comment:
```suggestion
  /// Evaluate \param E and if it can be evaluated to a siring literal,
```

Reply:
I am wondering is "string literal" the correct term to be using here? It seems it would be more correct to say something that can be evaluated to a null terminated string. 

A string literal can never not be null-terminated.